### PR TITLE
DONE: Param-Auction-Price-Function and Param-Auction-Price-Multiplier

### PR DESCRIPTION
**Pull request by Gala-G**
update

### CODE REVIEW

**Comment by Gala-G** [comment]
Staging links:
https://longforwisdom.gitbook.io/new-collection/parameter-index/collateral-auction/param-auction-price-function
https://longforwisdom.gitbook.io/new-collection/parameter-index/collateral-auction/param-auction-price-multiplier

File: parameter-index/collateral-auction/param-auction-price-function.md [hunk 1]
```diff
@@ -35,25 +35,24 @@ Controls the length of time between price drops. A smaller step means a smoother
...
+In an auction utilizing a Linear Decrease Auction Price Function, the auction will start at a price defined by the Auction Price Multiplier. The price will decrease linearly over time until it reaches zero. The time that it takes for an auction to reach zero is controlled by the `tau` function.
 
 #### tau
-A value in seconds which controls how long it takes for the price of a Linear Decrease auction to reach 0. This controls the rate of fall in the price of the collateral being auctioned. A higher tau will result in slower reduction in the price of collaterals, whereas a lower tau will cause faster falls in price.
+A value expressed in seconds which controls how long it takes for the price of a Linear Decrease auction to reach zero. This controls the rate of fall in the price of the collateral being auctioned. A higher `tau` will result in a slower reduction in the price of collaterals, whereas a lower `tau` will cause faster falls in price.
```
Comment (blimpa) [inline parameter-index/collateral-auction/param-auction-price-function.md:41]: So is tau a function or a parameter? Sounds like it's a parameter here, but we just said it's a function

File: parameter-index/collateral-auction/param-auction-price-function.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 >**Alias:** Price Curve  
 >**Parameter Name:** `calc`  
->**Containing Contract:** `Clipper`  
+>**Containing Contract:** `MCD_CLIP_$ILK`
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-function.md:5]: I would also change the key here from 'Containing Contract' -> 'Containing Contracts'. Uniquely (I think?) among our parameters, there are multiple contacts (one per ilk) as opposed to a single contract with a mapping of ilks->parameters.

File: parameter-index/collateral-auction/param-auction-price-function.md [hunk 3]
```diff
@@ -35,25 +35,24 @@ Controls the length of time between price drops. A smaller step means a smoother
 
 ### Linear Decrease
 
-In an auction utilizing a Linear Decrease Auction Price Function the auction will start at a price defined by the Auction Price Multiplier. The price will decrease linearly over time until it reaches 0. The time that it takes for an auction to reach 0 is controlled by the tau function.
+In an auction utilizing a Linear Decrease Auction Price Function, the auction will start at a price defined by the Auction Price Multiplier. The price will decrease linearly over time until it reaches zero. The time that it takes for an auction to reach zero is controlled by the `tau` function.
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-function.md:38]: `tau` is a parameter here, not a function.

File: parameter-index/collateral-auction/param-auction-price-function.md (hunk 1 above)
Comment (LongForWisdom) [reply parameter-index/collateral-auction/param-auction-price-function.md:41]: Hah, good catch also Pablo. Yes, I believe it's a parameter.

File: parameter-index/collateral-auction/param-auction-price-function.md [hunk 4]
```diff
@@ -35,25 +35,24 @@ Controls the length of time between price drops. A smaller step means a smoother
...
+A value expressed in seconds which controls how long it takes for the price of a Linear Decrease auction to reach zero. This controls the rate of fall in the price of the collateral being auctioned. A higher `tau` will result in a slower reduction in the price of collaterals, whereas a lower `tau` will cause faster falls in price.
 
-The Linear Decrease function is also bounded by the Maximum Auction Duration parameter. This can be used to set a lower bound on the price accepted by the Maker Protocol. Once this value has been reached, the price will stop falling, Keepers can re-start the auction from the initial price at this point.
+The Linear Decrease function is also bounded by the [Maximum Auction Duration parameter](param-max-auction-duration.md). This can be used to set a lower bound on the price accepted by the Maker Protocol. Once this value has been reached, the price will stop falling. Keepers can restart the auction from the initial price at this point.
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-function.md:43]: Minor thing, but probably just hyperlink the parameter name. So [Maximum Auction Duration parameter]. -> [Maximum Auction Duration] parameter.

File: parameter-index/collateral-auction/param-auction-price-function.md [hunk 5]
```diff
@@ -35,25 +35,24 @@ Controls the length of time between price drops. A smaller step means a smoother
...
-An auction price falling too slowly may increase the auction slippage, and could potentially cause bad debt.
+An auction price falling too slowly may increase the auction slippage and could potentially cause bad debt.
 
 ## Changes
 
-Adjusting an Auction Price Function parameter is a manual process that requires an executive vote. Changes to an Auction Price Function are subject to the GSM Pause Delay.
-
->Page last reviewed: -  
->Next review due: -  
+Adjusting an Auction Price Function parameter is a manual process that requires an executive vote. Changes to an Auction Price Function are subject to the [GSM Pause Delay](param-gsm-pause-delay.md).
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-function.md:55]: Check this link works on the staging site :)

File: parameter-index/collateral-auction/param-auction-price-multiplier.md [hunk 6]
```diff
@@ -2,54 +2,53 @@
 
 >**Alias:**  
 >**Parameter Name:** `buf`  
->**Containing Contract:** `Clipper`  
+>**Containing Contract:** `MCD_CLIP_$ILK`
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-multiplier.md:5]: Again, imo Contract -> Contracts

File: parameter-index/collateral-auction/param-auction-price-multiplier.md [hunk 7]
```diff
@@ -2,54 +2,53 @@
...
 
-If a user is liquidated and the OSM \(Oracle Security Module\) price is $100 and the Auction Price Multiplier is set to 1.6x, the starting auction price would be $160.
+For example, if a user is liquidated and the OSM (Oracle Security Module) price is $100 and the Auction Price Multiplier is set to 1.6x, the starting auction price would be $160.
 
 ## Purpose
 
-The primary purpose of the Auction Price Multiplier is so that when falling price collateral auctions begin the Maker Protocol does not begin the auction lower than the current market price in the general case. An ideal outcome for the Maker Protocol is for collateral auctions to start slightly above the market price and then fall to the market price as quickly as possible.
+The primary purpose of the Auction Price Multiplier is so that, when falling price collateral auctions begin, the Maker Protocol does not begin the auction lower than the current market price in the general case. An ideal outcome for the Maker Protocol is for collateral auctions to start slightly above the market price and then fall to the market price as quickly as possible.
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-multiplier.md:20]: Feels like this should be reworded some? Dunno, the first sentence sounds pretty awkward.

File: parameter-index/collateral-auction/param-auction-price-multiplier.md [hunk 8]
```diff
@@ -2,54 +2,53 @@
...
 
-This negative effect of the Auction Price Multiplier can be mitigated by setting an Auction Price Function that compliments the multiplier. For example by setting a higher Auction Price Multiplier in combination with an exponentially decreasing Auction Price Function. For more details on the Auction Price Function, see the appropriate documentation.
+This negative effect of the Auction Price Multiplier can be mitigated by setting an [Auction Price Function](param-auction-price-function.md) that complements the multiplier. For example, by setting a higher Auction Price Multiplier in combination with an exponentially decreasing Auction Price Function.
 
 ## Changes
 
-Adjusting an Auction Price Multiplier parameter for a specific vault type is a manual process that requires an executive vote. Changes to Auction Price Multiplier parameters are subject to the GSM Pause Delay.
+Adjusting an Auction Price Multiplier parameter for a specific vault type is a manual process that requires an executive vote. Changes to Auction Price Multiplier parameters are subject to the [GSM Pause Delay](param-gsm-pause-delay.md).
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-multiplier.md:39]: Check this link too.

File: parameter-index/collateral-auction/param-auction-price-multiplier.md [hunk 9]
```diff
@@ -2,54 +2,53 @@
...
 
 **Why increase this parameter?**
 
 An Auction Price Multiplier parameter might be increased for a specific vault type if in the general case auctions were starting at prices below the market price.
 
 **Why decrease this parameter?**
 
-An Auction Price Multiplier parameter might be decreased for a specific vault type if in the general case auctions were being regularly reset due to the price reaching the Max Auction Drawdown price.
+An Auction Price Multiplier parameter might be decreased for a specific vault type if in the general case auctions were being regularly reset due to the price reaching the [Max Auction Drawdown](max-auction-drawdown.md) price.
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-multiplier.md:47]: And check this one.

File: parameter-index/collateral-auction/param-auction-price-function.md [hunk 10]
```diff
@@ -35,25 +35,24 @@ Controls the length of time between price drops. A smaller step means a smoother
...
 
+>Page last reviewed: 11-07-2022
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-function.md:57]: YYYY-MM-DD

File: parameter-index/collateral-auction/param-auction-price-multiplier.md [hunk 11]
```diff
@@ -2,54 +2,53 @@
...
 
 ## Considerations
 
-The Starting Price for an auction is calculated using the OSM \(Oracle Security Module\) price feed multiplied by the Auction Price Multiplier.
-
->Page last reviewed: -  
->Next review due: -  
+The Starting Price for an auction is calculated using the OSM (Oracle Security Module) price feed multiplied by the Auction Price Multiplier.
 
+>Page last reviewed: 11-07-2022
```
Comment (LongForWisdom) [inline parameter-index/collateral-auction/param-auction-price-multiplier.md:53]: YYYY-MM-DD

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
Thanks Gala, see comments.

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**